packages/components/src/internal/components/auditlog/AuditQueriesListingPage.tsx

```diff
@@ -2,276 +2,202 @@
-        if (!this.hasDetailView()) return;
-
-        const model = this.getQueryModel();
-        if (!model) return;
 
-        // if the model has already loaded selections, we can use that to reselect the last row
-        if (!model.isLoadingSelections) {
-            this.updateSelectedRowId(this.getLastSelectedId());
-        }
-    };
-
-    getLastSelectedId = (): number => {
-        const model = this.getQueryModel();
-        const selectedIds = model.selections;
-        return selectedIds.size > 0 ? parseInt(Array.from(selectedIds).pop()) : undefined;
-    };
-
-    updateSelectedRowId = (selectedRowId: number): void => {
-        const { selected } = this.state;
-
-        if (this.state.selectedRowId !== selectedRowId) {
-            this.setState({ selectedRowId, detail: undefined }, async () => {
-                if (selectedRowId) {
-                    try {
-                        const auditEventType = selected === 'sourcesauditevent' ? 'queryupdateauditevent' : selected;
-                        const detail = await getAuditDetail(selectedRowId, auditEventType);
-
-                        this.setState({
-                            detail: detail.merge({ rowId: selectedRowId }) as AuditDetailsModel,
-                        });
-                    } catch (error) {
-                        console.error(error);
-                        this.setState({ error });
-                    }
-                }
-            });
-        }
-    };
-
-    get selectedQuery(): AuditQuery {
-        const { auditQueries } = this.state;
-
-        return auditQueries.find(q => q.value === this.state.selected);
-    }
-
-    get containerFilter(): Query.ContainerFilter {
-        return this.selectedQuery?.containerFilter;
-    }
-
-    hasDetailView(): boolean {
-        return this.state.selected && this.selectedQuery?.hasDetail === true;
-    }
-
-    getQueryModel = (): QueryModel => {
-        const { selected } = this.state;
-        if (!selected) {
-            return null;
-        }
-
-        const { queryModels, actions } = this.props;
-        const id = 'audit-log-querymodel-' + selected;
-
-        // only bind first model to URL so that it can pick up any filters passed from the caller
-        const isFirstModel = Object.keys(queryModels).length === 0;
-
-        if (!queryModels[id]) {
             actions.addModel(
                 {
+                    baseFilters,
+                    // only bind first model to URL so that it can pick up any filters passed from the caller
+                    bindURL: Object.keys(queryModels).length === 0,
+                    containerFilter: selectedQuery.containerFilter,
                     id,
-                    schemaQuery: new SchemaQuery(SCHEMAS.AUDIT_TABLES.SCHEMA, selected),
-                    containerFilter: this.containerFilter,
-                    bindURL: isFirstModel,
                     includeTotalCount: true,
-                    useSavedSettings: true,
```

Comment:
I believe this was just added by Alan last week. Just checking that this was intentionally removed?